src/osd/osd_types.h

```diff
@@ -4370,6 +4370,7 @@ struct store_statfs_t
   int64_t compressed = 0;              // Bytes stored after compression
   int64_t compressed_allocated = 0;    // Bytes allocated for compressed data
   int64_t compressed_original = 0;     // Bytes that were successfully compressed
+  int64_t garbage_collected = 0;       // Bytes written to recover space trapped because of overlapped blobs.
```

Comment:
Perhaps it's preferable to have such counter(s) implemented via perf counters. statfs structure is rather for generic storage space utilization statistics collection only. IMO we shouldn't overburden it. The counter we are currently discussing is an auxiliary one and hence doesn't deserve to be that common.